src/Action/SearchAction.php

```diff
@@ -78,7 +96,18 @@ public function __construct(
      */
     public function __invoke(Request $request): Response
     {
-        if (!$request->get('admin') || !$request->isXmlHttpRequest()) {
+        // NEXT_MAJOR: Remove this BC-layer.
+        if (null === $request->get('_sonata_admin') && null !== $request->get('admin')) {
```

Comment:
I never find in the code where the `admin` value is passed to the request.

Any idea @sonata-project/contributors ? I need to pass `_sonata_admin` instead to fix the deprecation on 3.x.

Reply:
hmm looks like it is supposed to be here:

https://github.com/sonata-project/SonataAdminBundle/blob/5e9b18a0fd548b1b26d312da0c2d70d0df0e7dcd/src/Resources/views/standard_layout.html.twig#L201-L210

but it doesn't and apparently it wasn't added at the beginning: https://github.com/sonata-project/SonataAdminBundle/commit/53dd431540fd9173bda6cb03d53c80d4fb030cf7

So I would say it is something we've never used.

Reply:
I will try to deprecate it instead then.